custom/enikshay/reports/filters.py

```diff
@@ -0,0 +1,34 @@
+from django.core.urlresolvers import reverse
+
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.reports.filters.base import BaseMultipleOptionFilter
+
+
+class EnikshayLocationFilter(BaseMultipleOptionFilter):
```

Comment:
is there anything that makes this a custom filter as opposed to one that could be dropped into any report for locations?